sources/Delimiter.swift

```diff
@@ -1,9 +1,9 @@
 /// Separators scalars/strings.
 public enum Delimiter {
     /// The CSV pair of delimiters (field & row delimiters).
-    public typealias Pair = (field: Self.Field, row: Self.Row)
-    /// The CSV pair of delimiter in string format.
-    internal typealias RawPair = (field: [Unicode.Scalar], row: [Unicode.Scalar])
+    public typealias Pair = (field: Self.Field, row: Self.Row?)
```

Comment:
Better than making it optional, I would prefer to have a `static` property in `Row` defining standard CRLF or some name similar. This way, we leave the door open for the future to support other types of delimiters.